presto-docs/src/main/sphinx/connector/tablestore.rst

```diff
@@ -0,0 +1,74 @@
+====================
+Tablestore Connector
+====================
+
+Overview
+--------
+
+The Tablestore Connector allows access to Alibaba Tablestore data from Presto.
+This document describes how to setup the Tablestore Connector to run SQL queries against Alibaba Tablestore.
+
+Configuration
+-------------
+
+To configure the Tablestore connector, create a catalog properties file
+``etc/catalog/tablestore.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=tablestore
+    tablestore.endpoint=https://demo.cn-hangzhou.ots.aliyuncs.com
+    tablestore.access-key-id=demo
```

Comment:
is `demo` intentionally used here? or, we could use `id`, `key`, `instance` or other names

Reply:
done